ng-device-detector.js

```diff
@@ -37,6 +37,10 @@
                     var ua = $window.navigator.userAgent;
                     var deviceInfo = uaDeviceDetector.parseUserAgent(ua);
                     deviceInfo.parseUserAgent = uaDeviceDetector.parseUserAgent;
+                    deviceInfo.custom = customDetectors.reduce(function (custom, customDetector) {
```

Comment:
I would prefer using `uaDeviceDetector` to do that functionality.

Reply:
Ok, I understand. So you prefer to "pass" the regex to the underlying uaDeviceDetector? Does it use the same syntax?